LEVELS.md

```diff
@@ -54,7 +54,7 @@ Here are the maturity levels. This is what we use as maturity levels and you may
 |**[T014](catalog/test/T014.md)**| Requirement - Acceptance Test Mapping > 20%. |
 |**[B006](catalog/build_ci/B006.md)**| Actively Use Available Analysis Tools (80% sonar-way). |
 |**[B007](catalog/build_ci/B007.md)**| Build Environment Can Be Automatically Created From Version Control. |
-|**[B008](catalog/build_ci/B008.md)**| CI/CD agents installation and config should be in version control. |
+|**[B008](catalog/build_ci/B008.md)**| No manual configuration of CI/CD agent machines. |
```

Comment:
Does this sentence cover the action of existing configuration in version control?
Because it could be automatic but there is no script in VC.
Am I right?

Reply:
No, it's on the item [B001](https://maturity.qualitic.ir/catalog/build_ci/B001.html). Since it's a kind of _CI config_.  
Plus the description already denotes it.